app/controllers/order.controller.js

```diff
@@ -2,24 +2,70 @@ const Order = require("../models/order.model.js");
 const Item = require("../models/item.model.js");
 // Create and Save a new order
 exports.create = (req, res) => {
+  // console.log("**********");
+  // console.log(req.body.customerName);
   // Validate request
   if (!req.body.items) {
     return res.status(400).send({
       message: "Order content can not be empty"
     });
   }
-  console.log(req.body.items);
```

Comment:
please use a logger and get rid of console logs

Reply:
Got it